internal/config/sqlx_gen_test.go

```diff
@@ -534,7 +534,7 @@ configs:
         path: gen/mysql/models
 `
 
-	filepath := path.Join(workDir, "sqlxgen.yaml")
+	filepath := path.Join(workDir, "sqlxgen.yml")
```

Comment:
it should be `yaml`

Reply:
The test was failing looking for the yaml file, as it was created as yml. Changing it to yaml let the test pass.

Reply:
yeah i need take a look at it, but for now i want the test cases to pass